x-pack/plugins/security_solution/common/license/policy_config.ts

```diff
@@ -269,9 +283,12 @@ export const unsetPolicyFeaturesAccordingToLicenseLevel = (
   policy: PolicyConfig,
   license: ILicense | null
 ): PolicyConfig => {
-  if (isAtLeast(license, 'platinum')) {
+  if (isAtLeast(license, 'enterprise')) {
```

Comment:
See my comment here: https://github.com/elastic/kibana/pull/168113/files#r1348940405

I think it will be cleanest to leave this logic alone and create another function to get the `Enterprise` features as @dasansol92 suggested.